src/System.Private.CoreLib/shared/System/Activator.RuntimeType.cs

```diff
@@ -149,46 +129,12 @@ public static ObjectHandle CreateInstance(string assemblyName, string typeName,
 
             if (type == null)
             {                
-                type = assembly.GetType(typeName, true /*throwOnError*/, ignoreCase);
+                type = assembly.GetType(typeName, throwOnError: true, ignoreCase);
             }
 
-            object o = Activator.CreateInstance(type,
-                                                bindingAttr,
-                                                binder,
-                                                args,
-                                                culture,
-                                                activationAttributes);
+            object o = CreateInstance(type, bindingAttr, binder, args, culture, activationAttributes);
 
-            return (o != null) ? new ObjectHandle(o) : null;          
-        }
-
-        public static ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName)
-        {
-            return CreateInstanceFrom(assemblyFile, typeName, null);
-        }
-
-        public static ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName, bool ignoreCase, BindingFlags bindingAttr, Binder binder, object[] args, CultureInfo culture, object[] activationAttributes)
-        {
-            return CreateInstanceFromInternal(assemblyFile,
-                                              typeName,
-                                              ignoreCase,
-                                              bindingAttr,
-                                              binder,
-                                              args,
-                                              culture,
-                                              activationAttributes);
-        }
-
-        public static ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName, object[] activationAttributes)
-        {
-            return CreateInstanceFrom(assemblyFile,
-                                      typeName,
-                                      false,
-                                      Activator.ConstructorDefault,
-                                      null,
-                                      null,
-                                      null,
-                                      activationAttributes);
+            return o != null ? new ObjectHandle(o) : null;          
         }
 
         private static ObjectHandle CreateInstanceFromInternal(string assemblyFile,
```

Comment:
This does not use RuntimeType. Can it be in `Activator.cs` ?

Reply:
It could be but it's using `assembly.GetType` which I don't know if works for corert either

Reply:
Both `Assembly.LoadFrom` and `Type.GetType` are public methods, so both of them are present in CoreRT.

(`Assembly.LoadFrom` throws PNSE in CoreRT, but that's fine.)